docs/platform/what-is-tizen/versions/tizen-5-0-m2.md

```diff
@@ -0,0 +1,45 @@
+# Tizen 5.0
```

Comment:
Kindly add file to ToC.

Reply:
I will raise the new PR after finishing "add tizen5.0 for what is tizen" and "add tizen5.5 for what is tizen"